src/components/ui/UiTimeline.story.vue

```diff
@@ -0,0 +1,74 @@
+<script lang="ts" setup>
+import UiTimeline from './UiTimeline.vue'
+
+function initState() {
+  return {
+    keydates: [
+      {
+        name: 'Semestre 1',
```

Comment:
I also prefer Data, but Piere told me that the timeline will be a year and a semester so I made this change https://github.com/camptocamp/rennes3d-front/pull/32/commits/4aacf8928c76b2759248249cac9e8fd3e6ae69a5. I also use Date under the hood to do the styling. I can check it again when I do the synchronizing between time line and the style.

Reply:
I have added a setDate function, so it's possible to set the state using Date: https://github.com/camptocamp/rennes3d-front/pull/32/commits/f22c0baa1bed80ec46f295e000f9b1e5852a0e82

Reply:
Yes, i told Ismail to use year and semester, because it's the only information we have. More over, in different portions, we have the exact same date for 2 or 3 different states, witch means that during the same semester the work will be done. If you put real date, it means during the same day, you will do the work, witch is impossible. One more thing, Rennes Métropole do not want to communicate throw real date, because there is alway delay in construction work. it is more comunicable throw semester.